core/src/main/java/org/axonframework/eventsourcing/eventstore/jdbc/JdbcEventStorageEngine.java

```diff
@@ -266,7 +269,6 @@ protected void storeSnapshot(DomainEventMessage<?> snapshot, Serializer serializ
      * @param snapshot   The snapshot to append.
      * @param serializer The serializer that should be used when serializing the event's payload and metadata.
      * @return A {@link PreparedStatement} that appends the snapshot when executed.
-     *
```

Comment:
My style settings do add these blank lines, and I do believe that my style guide is based on the guide tied to the Axon Framework. Should it be updated?